src/activemasternode.cpp

```diff
@@ -75,6 +67,7 @@ void CActiveMasternode::ManageStatus()
         }
 
         // The service needs the correct default port to work properly
+        std::string errorMessage;
         if (!CMasternodeBroadcast::CheckDefaultPort(service, errorMessage, "CActiveMasternode::ManageStatus()"))
             return;
```

Comment:
We should probably  update `notCapableReason` (including `errorMessage` in it) if this check fails.
No need to log, as  `CheckDefaultPort` takes care of logging failures (only with BCLog::MASTERNODE though, so maybe better to change it to `LogPrintf`).

Reply:
added the `notCapableReason` update.